<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ASCII</class>
 <widget class="QMainWindow" name="ASCII">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>657</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1142</width>
    <height>657</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1142</width>
    <height>657</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ASCII Creator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Logo.png</normaloff>Logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="Logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="import_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>İmport İmage File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Convert_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert to ASCII İmage File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save İmage File</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="width_new">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New Width :</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Your İmage</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ascii_output">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>891</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
